test/HcalDigiPipelineTest.cxx

```diff
@@ -271,7 +271,8 @@ class HcalCheckReconstruction : public framework::Analyzer {
     }
 
     // define target energy by using the settings at the top
-    double daq_energy{hit.getEnergy()};
+    double sampling_fraction = 0.11;
+    double daq_energy{hit.getEnergy()*sampling_fraction};
```

Comment:
With your current code for the producer, multiplying by the sampling fraction shouldn't be necessary because you don't divide by it.